CONTRIBUTING.md

```diff
@@ -2,74 +2,44 @@
 Contribution Guidelines
 =======================
 
-**Core ML Open Source Community**
+The Core ML `.mlmodel` file format is a publicly documented specification. The Core ML Tools source code is 100% open source under the [BSD license](https://en.wikipedia.org/wiki/BSD_licenses). We welcome all contributions and ideas to grow the product. We ask that you follow the [contributing guidelines and code of conduct](#guidelines-and-code-of-conduct)  in this document, which are typical of open source communities.
```

Comment:
There are a few different forms of the BSD License (ex three clause vs four clause, etc). Rather than linking to the general wikipedia page for BSD licenses. I'd recommend just linking to our license text file: https://github.com/apple/coremltools/blob/master/LICENSE.txt

Reply:
OK done